src/index.ts

```diff
@@ -335,7 +335,9 @@ export class GraphQLServer {
         subscriptionsEndpoint: this.subscriptionServerOptions
           ? this.subscriptionServerOptions.path
           : undefined,
-        tabs: this.options.defaultPlaygroundQuery
+        tabs: this.options.defaultPlaygroundTabs 
+        ? this.options.defaultPlaygroundTabs 
+        : this.options.defaultPlaygroundQuery
```

Comment:
I believe that you should update your else statement with [that](https://github.com/prisma/graphql-yoga/blob/2e174cb61384473d313ea57c304ebd853fbc1d40/src/index.ts#L339)

Reply:
Thanks for the comment @thawankeane!
The else statement does match the original `defaultPlaygroundQuery` ternary condition. 

``` js
        tabs: this.options.defaultPlaygroundTabs 
        ? this.options.defaultPlaygroundTabs 
        : this.options.defaultPlaygroundQuery
          ? [
              {
                endpoint: this.options.endpoint,
                query: this.options.defaultPlaygroundQuery,
              },
            ]
          : undefined,
```

or maybe you mean something else?

Reply:
Oh I'm sorry, I didn't notice the chained ternary